### DISCUSSION TITLE
China can not visit the base api url

### DISCUSSION THREAD

**Discussion by kobeontheway** [timdorr/tesla-api#660] [General]
at BeiJing city, when visit https://owner-api.vn.cloud.tesla.cn/ or https://owner-api.teslamotors.com/ over chrome browser, the page  said : Page not found

**Comment by core-hacked** [comment]
@kobeontheway Same response for me, it's not an error, you can still request the api as described in the docs.

**Reply by kobeontheway** [reply]
> @kobeontheway Same response for me, it's not an error, you can still request the api as described in the docs.

thx

**Reply by core-hacked** [reply]
@kobeontheway Can you also mark the answer as accepted, so people know that this was resolved?
Thx!
